.travis.yml

```diff
@@ -30,9 +30,7 @@ before_install:
   - pip install -U pip wheel setuptools
   - pip install -U $DJANGO
   - pip install --no-cache-dir -U -r requirements-test.txt
-  - ./runflake8
-  - ./runcheckmigration
-  - ./runisort
+  - openwisp-utils-qa-checks --migration-path "./django_loci/migrations ./django_loci/tests/testdeviceapp/migrations" --skip-checkmakemigrations
```

Comment:
please use a multiline statement for better readability, ensure you do this in all similar PRs